src/skmultiflow/meta/learn_pp.py

```diff
@@ -61,6 +61,19 @@ class LearnPPClassifier(BaseSKMObject, ClassifierMixin, MetaEstimatorMixin):
         'classes' parameter is not passed in the first partial_fit call, or if they
         are passed in further calls but differ from the initial classes.
 
+    Notes
+    -----
+        Originally, this ensemble algorithm was designed to train all of its members and combine
```

Comment:
After re-reading this in the context of the docstring I suggest we avoid mentioning code details and focus on the algorithmic aspects. Suggested rewording:

```python
    """
    Notes
    -----
    Originally, Learn++ is designed to train all of its members and combine
    their predictions considering the observed normalized errors. However,
    when training the base estimators, if the observed prediction error
    shrinks to zero before all estimators are trained, the error normalization
    is ill-defined, i.e., the instance error-based weight normalization factor
    (the sum of the errors) is zero.
    This implementation adds an 'early stop' mechanism to circumvent this
    corner case: LearnPPClassifier stops adding members to the ensemble if
    all instances are correctly classified. Otherwise (its normal behavior),
    the model uses as many ensemble members as defined via the ``n_estimators``
    parameter.
    """
```

Reply:
Thank you for the suggestion @jacobmontiel. It is more concise and avoid too many implemention details. I updated the description in commit [f37bb96](https://github.com/scikit-multiflow/scikit-multiflow/pull/223/commits/f37bb967d5f08874b51eab805ab8a7fff4d56a28).